hawkular-alerts-rest/src/main/java/org/hawkular/alerts/rest/TriggersHandler.java

```diff
@@ -451,6 +457,9 @@ public Response updateTrigger(
 
         } catch (Exception e) {
             log.debug(e.getMessage(), e);
+            if (e.getCause() != null && e.getCause() instanceof IllegalArgumentException) {
+                return ResponseUtil.badRequest("Bad arguments: " + e.getMessage());
+            }
```

Comment:
I know it may be tedious, but are there other services where we should now be returning "Bad arguments" for consistency?